ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.h

```diff
@@ -100,7 +97,9 @@ private slots:
   bool hasErrorOccurred(const Esri::ArcGISRuntime::Error& error);
   void addUtilityNetworkToMap(const Esri::ArcGISRuntime::Error& error);
   void setBusyIndicator(bool status);
-
+  void onIdentifyLayersCompleted_(const QList<Esri::ArcGISRuntime::IdentifyLayerResult*>& results);
+  void onTraceCompleted_();
+  void onTaskFailed(const Esri::ArcGISRuntime::ErrorException& exception);
```

Comment:
```suggestion
  void onTaskFailed_(const Esri::ArcGISRuntime::ErrorException& exception);
```

Since this is new, might as well follow the consistent pattern of trailing _ for private/protected methods.